jwst/pathloss/path_loss.py

```diff
@@ -0,0 +1,251 @@
+from __future__ import division
+
+#
+#  Module for calculating pathloss correction for science data sets
+#
+
+import numpy as np
+import logging
+from .. import datamodels
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
+
+def getCenter(input_model):
+    """
+    Get the center of the target in the aperture.
+    (0.0, 0.0) is the aperture center.  Coordinates go
+    from -0.5 to 0.5.
+    """
+    return (0.0, 0.0)
```

Comment:
I assume this is hardwired for now to return a centered location due to lack of available information that gives the actual source position? For MSA mode, the Shutter info table in the MSA configuration file contains entries giving the predicted x/y source centering within a given shutter from the MSA planning tool in APT. Those values could (and should) be loaded and stored with the rest of the slit meta data by the extract_2d step when it creates each of the extracted slits. Issue #212 includes directions to add source meta data from the MSA config file during the extract_2d step. That issue can be expanded to include reading the x/y centering entries from the shutter table and including them in the slit meta data as well. This will also require defining 2 new slit meta data entries in multislit.schema.yaml for these items. The names of the entries in the MSA config file Shutter table are "estimated_source_in_shutter_x" and "estimated_source_in_shutter_y" and I believe the values are already in the appropriate form where 0.0 represents perfectly centered and the range is -0.5 to +0.5. There can be multiple shutter entries in the Shutter table for a given slitlet_id (e.g. for a slitlet that has dimensions of 1x3 shutters), so the shutter that has 'background=N' (i.e. it contains the source) for a given slitlet_id is the one from which the x/y center values should be copied.